sdk/python/.pylintrc

```diff
@@ -148,6 +148,8 @@ disable=print-statement,
         invalid-name,
         protected-access,
         too-many-arguments,
+        too-many-branches,
```

Comment:
As much as it pains me to ignore these lints, code in `runtime/resource.py` often trips these up and it feels like more of a pain to refactor into single-use methods then it is to just document the functions aggressively.